fastlane/lib/fastlane/setup/setup_ios.rb

```diff
@@ -34,6 +34,8 @@ def run(user: nil)
           detect_if_app_is_available
         end
         print_config_table
+        UI.important("Note: If the values above are not correct, it is possible the wrong scheme was selected by fastlane")
+        UI.important("As a workaround, you can drag your scheme to the top of the scheme editor in Xcode (Product > Scheme > Manage Schemes...)")
```

Comment:
Maybe this edge case should "only" be documented in the docs and these two lines be replaced by URL? This is shown to _all_ `fastlane init` users.

Reply:
Huh, it seems something was messed up there. I previously did a force push since I thought of the same thing, so now it only shows it to people with multiple schemes

Reply:
I re-pushed, now it shows the correct code 👍🏼